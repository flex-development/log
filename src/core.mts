/**
 * @file Core
 * @module log/core
 */

export { default as logLevels } from '#enums/log-levels'
export * from '#lib/index'
