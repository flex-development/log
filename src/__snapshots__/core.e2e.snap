// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`e2e:log/core > should expose public api 1`] = `
[
  "logLevels",
  "createLogger",
]
`;
